assets/templates/kubernetes.container/schema-b/gotext_multiline.tpl

```diff
@@ -71,14 +74,14 @@
          "cpu":{
             "usage":{
                "core":{
-                  "ns": 41129679
+                  "ns": {{mul $usage_nanoseconds 1000}}
                },
                "node":{
-                  "pct": {{divf $pct 1000000}}
+                  "pct": {{divf $pctcpu 100}}
                },
-               "nanocores":0,
+               "nanocores":{{mul $usage_nanocores 1000}},
```

Comment:
`mul 1000` is required to have the proper order of magnitude with the proper resolution of trailing zeros correct?

Like: I want values between 100000000 and 9000000000 but with zeros as units, tens and hundreds, correct?

If you don't care for the trailing zeros you can just use 100000000 and 9000000000 in the range. It should not overflow if I'm not wrong.

Reply:
I just kept above suggestion and did not follow this as might not be clear for the outcome of the results.